defaults/main.yml

```diff
@@ -74,6 +77,11 @@ postgresql_pid_directory: "/var/run/postgresql"
 # If external_pid_file is not explicitly set, on extra PID file is written
 postgresql_external_pid_file: "{{ postgresql_pid_directory }}/{{postgresql_version}}-{{postgresql_cluster_name}}.pid"
 
+# Docker mode variables
+postgresql_data_docker_directory: "/var/lib/postgresql/data/pgdata"
+postgresql_hba_file_docker: "{{postgresql_data_docker_directory}}/pg_hba.conf"
```

Comment:
it should be possible without these 2 new vars.
I'ld rather keep (from the defaults):
```
postgresql_hba_file: "{{postgresql_conf_directory}}/pg_hba.conf"
```
and use set_fact (see example below)
plus new variables, to allow different folders (when deploying changes on the docker host):
```
postgresql_conf_directory_on_host
postgresql_data_directory_on_host
```